packages/generator-one-app-module/generators/app/templates/intl-child-module/__tests__/locale.spec.js

```diff
@@ -0,0 +1,43 @@
+import path from 'path';
+import { promisify } from 'util';
+
+const glob = promisify(require('glob'));
+
+const localeDir = path.join(__dirname, '../locale');
+
+const schema = {
+  properties: {
+    greeting: {
+      type: 'string',
+    },
+  },
+  required: ['greeting'],
+  additionalProperties: false,
+};
+
+describe('Language packs', () => {
+  let locales = null;
+  beforeAll(() => glob('*', { cwd: localeDir }).then((result) => {
+    locales = result;
+    return locales;
+  }));
+
+  it('should have the correct schema in each copy file', (done) => {
+    const failures = [];
+    locales.forEach((localeFile) => {
+      // eslint-disable-next-line global-require,import/no-dynamic-require
+      const copy = require(path.join(localeDir, localeFile));
+      try {
+        expect(copy).toMatchSchema(schema, localeFile);
```

Comment:
`toMatchSchema` comes from `jest-json-schema` which wont be brought in by amex-jest-preset-react once you update to the latest version. So you should add that dep and the jest-setup file to set it up in this module template.

Reply:
required and setup in test-setup file for intl templates (not needed in basic module templates, no toMatchSchema tests in there)